test/unit/channel.js

```diff
@@ -844,35 +844,3 @@ describe('Channel _initializeState', () => {
 		expect(Object.keys(channel.state.members)).deep.to.be.equal(['alice']);
 	});
 });
-
-describe('pending message', () => {
```

Comment:
This unit test didn't make sense anymore since entire credential check has been moved to backend.